dcos_launch/platforms/onprem.py

```diff
@@ -11,6 +11,7 @@
 
 log = logging.getLogger(__name__)
 
+NGINX_DOCKER_IMAGE_VERSION='nginx:1.15.2'
```

Comment:
This is the tox issue that needs to be addressed

```
[03:47:33]	./dcos_launch/platforms/onprem.py:14:27: E225 missing whitespace around operator
[03:47:33]	./dcos_launch/platforms/onprem.py:16:1: E302 expected 2 blank lines, found 1
```

In line Line 14, please add space around operator `=`
and add another line in line 15.
That will help us proceed with merging and building this.